cmd/sensors/src/lib.rs

```diff
@@ -133,13 +137,14 @@ fn list(
             }
             HubrisSensorDevice::Other(device, _) => {
                 println!(
-                    "{:2} {:7} {:2} {:2} {:3}   {:02} {:13} {:<1}",
+                    "{:3} 0x{:03x} {:7} {:2} {:2} {:3} {:02} {:13} {:<1}",
```

Comment:
```suggestion
                    "{:3} {:#05x} {:7} {:2} {:2} {:3} {:02} {:13} {:<1}",
```

Reply:
Great tip! And allows me to lose the leading zeroes, which were somewhat unsightly...